flagsmith-core.ts

```diff
@@ -37,8 +37,10 @@ type RequestOptions = {
 }
 
 let AsyncStorage: AsyncStorageType = null;
-const FLAGSMITH_KEY = "BULLET_TRAIN_DB";
-const FLAGSMITH_EVENT = "BULLET_TRAIN_EVENT";
+const DEFAULT_FLAGSMITH_KEY = "BULLET_TRAIN_DB";
```

Comment:
Let's (finally) change this to FLAGSMITH!

Reply:
resolved